WrightTools/tuning/workup.py

```diff
@@ -178,8 +180,16 @@ def tune_test(data, curve, channel_name, level=False, cutoff_factor=0.01,
     yi = outs.mean.values
     spline = wt_kit.Spline(xi, yi)
     offsets_splined = spline(xi)  # wn
-    # make curve ----------------------------------------------------------------------------------
+    # check old curve for compatability with data object make curve -------------------------------
     curve = curve.copy()
+    try:
+        assert np.array_equal(curve.colors, outs.axes[0].points)
```

Comment:
This line is not units aware, also assertions of equality are often dangerous with floating point numbers in general, as a tolerance is preferred. Some floating point operations are are architecture dependent on rounding.